app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,47 @@
+class ReviewsController < ApplicationController
+  def new
+    @game = Game.find(params[:game_id])
+    @review = Review.new
+  end
+
+  def create
+    @game = Game.find(params[:game_id])
+    @review = Review.new(review_params)
+    @review.user_id = current_user.id
+    @review.game_id = params[:game_id]
+
+    if @review.save
+      flash[:notice] = "review submitted."
+      redirect_to "/games/#{@game.id}"
```

Comment:
write this the rails way (look at `rake routes` to see what path you'd write here instead).